src/ts/components/smallcenteredplaybacktogglebutton.ts

```diff
@@ -0,0 +1,121 @@
+import {PlaybackToggleButton, PlaybackToggleButtonConfig} from './playbacktogglebutton';
+import {DOM} from '../dom';
+import {UIInstanceManager} from '../uimanager';
+import { PlayerAPI, PlayerEventBase, WarningEvent } from 'bitmovin-player';
+import { i18n } from '../localization/i18n';
+
+export class SmallCenteredPlaybackToggleButton extends PlaybackToggleButton {
+
+  constructor(config: PlaybackToggleButtonConfig = {}) {
+    super(config);
+
+    this.config = this.mergeConfig(config, {
+      cssClass: 'ui-smallcenteredplaybacktogglebutton',
+      text: i18n.getLocalizer('playPause'),
+      role: 'button',
+    }, this.config);
+  }
+
+  configure(player: PlayerAPI, uimanager: UIInstanceManager): void {
+    // Update button state through API events
+    super.configure(player, uimanager, false);
+
+    // Set enterFullscreenOnInitialPlayback if set in the uimanager config
+    if (typeof uimanager.getConfig().enterFullscreenOnInitialPlayback === 'boolean') {
+      this.config.enterFullscreenOnInitialPlayback = uimanager.getConfig().enterFullscreenOnInitialPlayback;
+    }
+
+    let togglePlayback = () => {
+      if (player.isPlaying() || this.isPlayInitiated) {
+        player.pause('ui');
+      } else {
+        player.play('ui');
+      }
+    };
+
+    let firstPlay = true;
+
+    this.onClick.subscribe(() => {
+      togglePlayback();
+
+      if (firstPlay && this.config.enterFullscreenOnInitialPlayback) {
+        player.setViewMode(player.exports.ViewMode.Fullscreen);
+      }
+    });
+
+    player.on(player.exports.PlayerEvent.Play, () => {
+      // Playback has really started, we can disable the flag to switch to normal toggle button handling
```

Comment:
```suggestion
    player.on(player.exports.PlayerEvent.Playing, () => {
      // Playback has really started, we can disable the flag to switch to normal toggle button handling
```
Play only signals the intent to start playback, not actual success.

Reply:
This was from the old `HugePlaybackToggleButton`. But yeah... makes sense